CMakeLists.txt

```diff
@@ -229,15 +237,15 @@ else()
 
     if(QUIC_SANITIZE_ADDRESS)
         message(STATUS "Configuring address sanitizer")
-        set(QUIC_COMMON_FLAGS "${QUIC_COMMON_FLAGS} -fsanitize=address -fno-omit-frame-pointer -Wno-maybe-uninitialized -O0 -Og -g")
```

Comment:
-Og actually does optimizations near -O1, so prefer -O0 over it.

Reply:
I'm fine with this. For asan, O0 should be fine.